packages/components/src/components/telekom/telekom-footer/telekom-footer-content.css

```diff
@@ -38,6 +41,10 @@ footer {
   display: flex;
   flex: 1;
   flex-direction: column;
+  width: 100%;
```

Comment:
I think this would be overwritten by the following width declarations, maybe it could be removed?